### ISSUE TITLE
Tab header foreground color not set properly

### ISSUE THREAD

**Issue by mikechristiansenvae** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2653]
### Issue

The `Foreground` property on `TabItem` does not seem to be set properly.

The attached screenshots show that the `Foreground` property is inherited correctly up until the `ColorZone`.  It appears that the `ColorZone` inherits a transparent foreground from its template.

Problem occurs for light mode and dark mode.

May be related to [PR #2544](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2544) ?

### Reproduction

[Reproduction project repository](https://gitlab.com/michael.christiansen/material_design_bug_demo)

`App.xaml`

```
<Application x:Class="WpfSample.App"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:local="clr-namespace:WpfSample"
             xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
             StartupUri="MainWindow.xaml">
    <Application.Resources>

        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <materialDesign:BundledTheme BaseTheme="Light" PrimaryColor="Blue" SecondaryColor="Amber" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>
</Application>
```

`MainWindow.xaml`

```
<Window x:Class="WpfSample.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfSample"
        mc:Ignorable="d"

        xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
        TextElement.Foreground="{DynamicResource MaterialDesignBody}"
        Background="{DynamicResource MaterialDesignPaper}"
        TextElement.FontWeight="Medium"
        TextElement.FontSize="14"
        FontFamily="{materialDesign:MaterialDesignFont}"

        Title="MainWindow" Height="450" Width="800">
    <TabControl TabStripPlacement="Left">
        <TabItem Header="Foo">
            <TextBlock Text="Foo Content" />
        </TabItem>
        <TabItem Header="Bar">
            <TextBlock Text="Bar Content" />
        </TabItem>
    </TabControl>
</Window>
```

### Screenshots

![image](https://user-images.githubusercontent.com/55414409/164515333-c92d68b1-e209-4f3b-905d-b8890b9bdcf5.png)

![image](https://user-images.githubusercontent.com/55414409/164515408-69069c4f-dc58-46b6-9827-0bc65f28c0c9.png)

![image](https://user-images.githubusercontent.com/55414409/164514407-f9afb3d2-e385-427d-a843-d9e1860a3066.png)

![image](https://user-images.githubusercontent.com/55414409/164514447-130ef0a6-5236-495c-8b03-e548e93aadfc.png)

**Comment by mikechristiansenvae** [comment]
[This may be a duplicate of issue 2602 ](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2602)

**Comment by mikechristiansenvae** [comment]
Issue seems to be resolved with nuget package `4.5.0-ci147`.  Closing issue.
